doc/nrf/ug_matter_hw_requirements.rst

```diff
@@ -162,3 +166,302 @@ Values are provided in kilobytes (KB).
       +------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+-------------+
 
 ..
+
+.. _ug_matter_hw_requirements_layouts:
+
+Matter memory layouts
```

Comment:
Should we call it "Reference Matter memory layouts"?
I mean we should somewhere make it clear that these are just layouts that are tested by Nordic, but a manufacturer should consciously tune the partition sizes according to their needs.